docs/hadoop-catalog.md

```diff
@@ -32,6 +32,7 @@ Besides the [common catalog properties](./gravitino-server-config.md#gravitino-c
 | `default-filesystem-provider`                      | The name default filesystem providers of this Hadoop catalog if users do not specify the scheme in the URI. Default value is `builtin-local`                                                                                                                                                                   | `builtin-local` | No                                                          | 0.7.0-incubating |
 | `authentication.impersonation-enable`              | Whether to enable impersonation for the Hadoop catalog.                                                                                                                                                                                                                                                        | `false`         | No                                                          | 0.5.1            |
 | `authentication.type`                              | The type of authentication for Hadoop catalog, currently we only support `kerberos`, `simple`.                                                                                                                                                                                                                 | `simple`        | No                                                          | 0.5.1            |
+| `java.security.krb5.conf`                          | Kerberos krb file for configuration of Kerberos.                                                                                                                                                                                                                                                              | /etc/krb5.conf  | No                                                          | 0.7.0-incubating |
```

Comment:
Let me think if there is a better name for it, `java.security.krb5.conf` is so long.

Reply:
what about `krb5.conf.path`?

Reply:
LGTM about the name changing. That mean I should create a new PR for it, is it correct?

Reply:
> That mean I should create a new PR for it?

No, if it's okay, please do it in this PR.